index.js

```diff
@@ -243,6 +243,17 @@ exports.stringify = (object, options) => {
 	}, options);
 
 	const formatter = encoderForArrayFormat(options);
+
+	const object = Object.assign({}, input);
+
+	if (options.skipNullAndUndefined) {
+		for (const key of Object.keys(object)) {
+			if (object[key] === undefined || object[key] === null) {
```

Comment:
It doesn't work with `arrayFormat`. You need to check if `object[key]` is an array, and iterate through array to check null/undefined.